packages/opentelemetry-exporter-collector-proto/src/CollectorExporterNodeBase.ts

```diff
@@ -40,6 +41,7 @@ export abstract class CollectorExporterNodeBase<
         _onFinish();
       };
       const _onError = (error: collectorTypes.CollectorExporterError): void => {
+        globalErrorHandler(error);
```

Comment:
Not sure i follow, currently the `onError` on the exporter collector also calls the exporter logger, shouldn't the `gloalErrorHandler` be in the `CollectorExporterBase` ?

Reply:
Good catch @vmarchaud. Since the globalErrorHandler is already used in `CollectorExporterBase` it doesn't need to be called from subclasses. I've updated.